src/ui/screens/Parachains/AuctionsScreen.tsx

```diff
@@ -35,6 +36,7 @@ export function AuctionsScreen() {
 
   const {auctionsInfo, latestAuction} = auction;
   const {winningBid, leasePeriod, endingPeriod, raised, raisedPercent} = latestAuction;
+  const raisedFormatted = formatBalance(raised) ?? '';
```

Comment:
You have to pass the `isShort` argument to get the short value.

```suggestion
  const raisedFormatted = formatBalance(raised, {isShort: true}) ?? '';
```